style.css

```diff
@@ -1,7 +1,65 @@
 *, *::before, *::after { box-sizing: border-box; }
 body {
   font-family:-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  font-size: 100%;
+  font-size: 112.5%;
   line-height: 1.5;
   margin: 0;
+  color: #293241;
+  background: #e0fbfc;
+}
+
+header h1,
+main,
+footer p {
+  margin: 0 auto;
+  max-width: 42rem;
+  padding: 1rem;
+}
+
+header,
+footer {
+  padding: 2rem 1rem;
+  background-color: #98c1d9;
+}
+
+main {
+  padding: 4rem 1rem;
+}
+
+:is(h2, h3, h4, h5, h6) {
+  margin: 4rem 0 1rem;
+  line-height: 1.2;
+}
+
+h1 {
+  font-size: 3rem;
+  margin: 0;
+}
+
+h2 {
+  font-size: 2.2rem;
+}
+
+h2:first-child {
+  margin-top: 0;
+}
+
+h3 {
+  font-size: 1.8rem;
+}
+
+h4 {
+  font-size: 1.4rem;
+}
+
+ul {
+  list-style-type: square;
+}
+
+li + li {
+  margin-top: 0.5rem;
+}
+
+::marker {
+  color: #ee6c4d;
```

Comment:
This will need to be tweaked for color-contrast: 

![image](https://user-images.githubusercontent.com/4587451/150177383-544a26d9-03c2-44f4-9b92-dd3c3d97bd64.png)

After that, I would be delighted to accept this PR!

Reply:
Is AA ok? I get a ~6.5:1 ratio, which passes everything except normal text AAA. I figure that's _probably_ ok since it's just bullets/numbering and not body copy, but we can just pull it out if not.